.github/workflows/lint.yml

```diff
@@ -0,0 +1,37 @@
+name: Create Block
+
+on: [pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Use Node.js 12.x
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12.x
+    - name: npm install, build
+      run:
+        - npm ci
+        - npm run test:create-block
+
+        # eslint-plugin-import/no-extraneous-dependencies relies on modules to
+        # be "installed", which in practice requires that packages be built,
+        # presumably so that pkg.main can be resolved.
+        #
+        # See: https://github.com/benmosher/eslint-plugin-import/blob/92caa35/resolvers/node/index.js
+        - node ./bin/packages/build.js
+    - name: npm install, build
+      run: |
+        npm ci
+        npm run test:create-block
+        npm run build
+    - name: Lint JavaScript and Styles
+      run: npm run lint
+    - name: Lint ES5 built files (IE11)
+      run: npx eslint --parser-options=ecmaVersion:5 --no-eslintrc --no-ignore ./build/**/*.js
+    - name: Type checking
+      run: npm run build:package-types
```

Comment:
I thought a single workflow for all linting tasks is better than separate ones. I might also add php linting later.

Reply:
PHP requires Docker or composer and all that jazz, it might better to combine it with PHP unit tests